react_app/src/components/MovieForm/MovieForm.jsx

```diff
@@ -24,51 +25,74 @@ const MOVIE_FORM = {
 };
 
 const MovieForm = ({ onSubmit, movie }) => {
+  const { register, handleSubmit, reset } = useForm({
+    defaultValues: {
+      ...movie,
+    },
+  });
```

Comment:
```suggestion
  const { register, handleSubmit, reset } = useForm();
```
You have an empty form because on first render `movie === {}`. 
You can handle this by adding `reset` after first render.
```js
  useEffect(() => {
    movie && reset(movie);
  }, [movie, reset]);
```

Or you could add [conditional rendering](https://github.com/TatsianaSlapik/reactJS/pull/7/files#diff-e593c483efbdfb579a774b1fc9b5be5579726bf3d43a4d6e19d82c75aae9cc1dR33) into `EditMovieForm.jsx`